### PR TITLE
Triggers: Remove HitBlock and BreakBlock

### PR DESCRIPTION
**Pull request by camnwalter**
Not really a point to have both these and a PlayerInteract, so just merged them all together

### CODE REVIEW

File: src/main/kotlin/com/chattriggers/ctjs/minecraft/listeners/ClientListener.kt [hunk 1]
```diff
@@ -193,73 +195,83 @@ object ClientListener : Initializer {
...
             val event = CancellableEvent()
 
+            val stack = player.getStackInHand(hand)
+
             TriggerType.PLAYER_INTERACT.triggerAll(
                 PlayerInteraction.USE_ITEM,
-                // TODO,
-                event
+                if (stack.isEmpty) null else Item(stack),
+                hand,
```
Comment (mattco98) [inline src/main/kotlin/com/chattriggers/ctjs/minecraft/listeners/ClientListener.kt:273]: Are any of the interaction events applicable to both hands? It doesn't seem like it but I could be wrong. That's why I originally had the field, so you could just check `type.isLeftHand` or whatever

File: src/main/kotlin/com/chattriggers/ctjs/engine/IRegister.kt [hunk 2]
```diff
@@ -461,6 +461,7 @@ interface IRegister {
      *   [com.chattriggers.ctjs.minecraft.wrappers.entity.Entity],
      *   [com.chattriggers.ctjs.minecraft.wrappers.world.block.Block], or
      *   [com.chattriggers.ctjs.minecraft.wrappers.inventory.Item],
+     *  - The Hand used
```
Comment (mattco98) [inline src/main/kotlin/com/chattriggers/ctjs/engine/IRegister.kt:464]: Extra space

File: src/main/kotlin/com/chattriggers/ctjs/minecraft/listeners/ClientListener.kt [hunk 3]
```diff
@@ -193,73 +195,83 @@ object ClientListener : Initializer {
...
+                CancellableEvent(),
             )
-
-            !event.isCancelled()
         }
 
-        UseBlockCallback.EVENT.register { _, _, _, hitResult ->
+        UseBlockCallback.EVENT.register { player, _, hand, hitResult ->
+            // TODO: double firing
```
Comment (mattco98) [inline src/main/kotlin/com/chattriggers/ctjs/minecraft/listeners/ClientListener.kt:236]: To clarify, it double fires event with the client check?

**Review by mattco98 — CHANGES_REQUESTED** [review]

File: src/main/kotlin/com/chattriggers/ctjs/minecraft/listeners/ClientListener.kt (hunk 3 above)
Comment (camnwalter) [reply src/main/kotlin/com/chattriggers/ctjs/minecraft/listeners/ClientListener.kt:236]: Oops. Meant to delete that comment. The double firing was from triggering on both the main hand and offhand.

File: src/main/kotlin/com/chattriggers/ctjs/minecraft/listeners/ClientListener.kt (hunk 1 above)
Comment (camnwalter) [reply src/main/kotlin/com/chattriggers/ctjs/minecraft/listeners/ClientListener.kt:273]: Yeah the UseBlock, UseEntity, and UseItem fire on both main and off hand.

**Pull request merged**